extensions/pdh/PerformanceDataMonitor.cpp

```diff
@@ -45,9 +45,13 @@ core::Property PerformanceDataMonitor::CustomPDHCounters(
 core::Property PerformanceDataMonitor::OutputFormatProperty(
     core::PropertyBuilder::createProperty("Output Format")->
     withDescription("Format of the created flowfiles")->
-    withAllowableValue<std::string>(JSON_FORMAT_STR)->
-    withAllowableValue(OPEN_TELEMETRY_FORMAT_STR)->
-    withDefaultValue(JSON_FORMAT_STR)->build());
+    withAllowableValues<std::string>({ PRETTY_JSON_FORMAT_STR, COMPACT_JSON_FORMAT_STR, PRETTY_OPEN_TELEMETRY_FORMAT_STR, COMPACT_OPEN_TELEMETRY_FORMAT_STR })->
```

Comment:
I would prefer two separate properties: `Output Format = JSON | OPEN_TELEMETRY` and `Output Style = PRETTY | COMPACT`, as that would make the code simpler, and I think the interface would be clearer, too.

Reply:
changed it